libqtile/widget/volume.py

```diff
@@ -185,11 +196,15 @@ def get_volume(self):
             if self.get_volume_command:
                 get_volume_cmd = self.get_volume_command
 
-            mixer_out = self.call_process(get_volume_cmd)
+            mixer_out = subprocess.getoutput(get_volume_cmd)
```

Comment:
What's the reason for this change?

Reply:
Thanks for sharing this.

It's an interesting scenario as `getoutput` doesn't return an error when it's passed a list but it seems that it just runs the first item in the list.

Do you want to submit a fix for this?

Reply:
I think just falling back to `self.call_process` would work. By the way, the current version "happens to work", because the output of the command intended to call, by default `amixer -D default sget Master`, is just part of the output of `amixer`, and the first `\d?\d?\d?%` happens to be the volume of `Master`.